src/H5FDros3.h

```diff
@@ -68,13 +68,21 @@
  *
  *     String: "Secret Access Key" associated with the ID and resource.
  *
+ * `secret_tok` (char[])
```

Comment:
Since `H5FD_ros3_fapl_t` doesnt contain this as a top-level field, this information should be moved to the declarations for `H5Pget_fapl_ros3_token`/`H5Pget_fapl_ros3_token`.

Reply:
Yes, you are right. Removed this description from the header file.